docs/networking/loadbalancer.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 4
+sidebar_label: Load Balancer
+title: "Load Balancer"
+keywords:
+- Load Balancer
+---
+_Available as of v1.2.0_
+
+The Harvester load balancer (LB) is a built-in Layer 4 load balancer that distributes incoming traffic across workloads deployed on Harvester virtual machines (VMs) or guest Kubernetes clusters.
+
+## VM load balancer
+
+### Features
+Harvester VM load balancer supports the following features:
+
+- **Address assignment:** Get the LB IP address from a DHCP server or a pre-defined IP pool.
+- **Protocol support:** Supports both TCP and UDP protocols for load balancing.
+- **Multiple listeners:** Create multiple listeners to handle incoming traffic on different ports or with other protocols.
+- **Label selector:** The LB uses label selectors to match the backend servers. Therefore, you must configure the corresponding labels for the backend VMs you want to add to the LB.
+- **Health check:** Only send traffic to healthy backend instances.
+ 
+### Limitations
+Harvester VM load balancer has the following limitations:
+
+- **Namespace restriction:** This restriction facilitates permission management and ensures the LB only uses VMs in the same namespace as the backend servers.
+- **IPv4-only:** The LB is only compatible with IPv4 addresses for VMs.
+- **Guest agent installation:** Installing the guest agent on each backend VM  is required to obtain IP addresses.
```

Comment:
Q: do we support Windows VM in this case?

Reply:
Let me check it.

Reply:
We are unable to install the guest agent for the Windows VM. Therefore, Windows VM is not supported.

Reply:
I'll add a note.